src/components/Table/GridTable.tsx

```diff
@@ -313,9 +334,9 @@ export function GridTable<R extends Kinded, X extends Only<GridTableXss, X> = an
     const expandableHeaderRows: RowTuple<R>[] = [];
     const totalsRows: RowTuple<R>[] = [];
     const visibleDataRows: RowTuple<R>[] = [];
+    const unmatchedSelectedRows: RowTuple<R>[] = [];
```

Comment:
I went with "unmatched" because rows can still be "hidden" if their parent is in a collapsed state. I wanted a distinction between rows that are hidden due to not matching the filter vs rows that are hidden due to their parent being collapsed.

Reply:
Makes sense... Still kinda thinking for a different term, like "pinned" (but we already used pinned), "offpage", "extra" ... "kept"? I kinda like "kept" b/c it denotes they've gone away but we've "kept" them. Wdyt?

Reply:
Sure. I can update to `kept`. 
Though, in `TableState` I might keep this naming as `unmatchedSelectedRows` as we also have another property called `matchedRows`:
```
// Set of just row ids. Keeps track of which rows match the filter. Used to filter rows from `selectedIds`
private matchedRows = new ObservableSet<string>();
```
Since what we are storing is basically the exact opposite of what this one keeps track of, then I went with `unmatched`.

Reply:
Hm, it's more than the "opposite of matchedRows" though? Like matched rows is "subset of props.rows that match the filter" (regardless of being selected/unselected?) and "kept rows" is "any row that was at-one-point a) in props.rows and b) selected".

Reply:
Yea, not the exact opposite. It is a list of rows that are selected AND no longer in the `matchedRows` set. This means it could be a row that is no longer in `props.rows`, or a row that doesn't match the existing client-side filter.

Reply:
It sounds like there may be a distinction there that went unnoticed, specifically where `unmatchedSelectedRows` can included selected rows that are no longer part of the `props.rows` and rows that do not matched the current filter supplied to `GridTable`

Reply:
I will go ahead and change everywhere I have `unmatched` to `kept`. I don't feel too strongly on the name, and `kept` is much shorter. Hopefully the comments will help define its purpose.